### PR TITLE
Event Model

### PR DESCRIPTION
**Pull request by tarungog**
Fixes # .
#268
### PR Description
.
### What needs to be reviewed?
So I'm not sure where I would make it easier to construct an event, as we're going to manually have to find the location object and feed in its _id member to the event.create() function.
### Context?
events for map back end

### CODE REVIEW

File: server/db/model/Event.js [hunk 1]
```diff
@@ -1,4 +1,73 @@
 /* eslint-disable */
 var mongoose = require('../index.js'),
     config = require('../../../config/default.js'),
-    Schema = mongoose.Schema;
+    Schema = mongoose.Schema
+Location = require('./Location.js');
```
Comment (connorkrupp) [inline server/db/model/Event.js]: Add location to the comma list of imports here, declaring a variable without var/let/const puts it in the global namespace

File: server/db/model/Event.js [hunk 2]
```diff
@@ -1,4 +1,73 @@
...
+        default: Date.now()
+    },
+    category: {
+        type: String,
+        enum: [
+            'General',
+            'Food',
+            'Tech Talk',
+            'Sponsor Event',
+            'we should get a config list for this'
```
Comment (connorkrupp) [inline server/db/model/Event.js]: @antoniomika what is the proper way to handle this mongo style? Or at least non-relational style

Comment (antoniomika) [inline server/db/model/Event.js]: @connorkrupp What do you mean exactly?

File: server/db/model/Event.js [hunk 3]
```diff
@@ -1,4 +1,73 @@
 /* eslint-disable */
```
Comment (antoniomika) [inline server/db/model/Event.js]: Enable ESLint now that the model is actually complete

**Pull request closed without merging**